sklearn/feature_extraction/text.py

```diff
@@ -306,6 +306,15 @@ def _check_vocabulary(self):
         if len(self.vocabulary_) == 0:
             raise ValueError("Vocabulary is empty")
 
+    def _validate_params(self):
+        """Check validity of ngram_range parameter"""
+        min_n, max_m = self.ngram_range
```

Comment:
Just a side note: I'm not certain whether we want to validate e.g. the `ngram_range` length, or `min_n`, `max_n` dtype. I think the current code might be OK for now.

Reply:
We should validate that min_n <= max_m